website/docs/time_off.md

```diff
@@ -238,7 +239,7 @@ Below is a summary of our paid leave programs.
 | Parental Leave    	| 16 Weeks                 	| Intermittent    	| Paid Family Leave,State,Unum    	|                                                                                   	|
 | Medical Leave     	| 4 Weeks                  	| Continuous      	| State,Unum                      	| Capped at 8 weeks if a team member is granted two medical leaves within one year. 	|
 | Caregiver Leave   	| 4 Weeks                  	| Intermittent    	| Paid Family Leave               	|                                                                                   	|
-| Miscarriage Leave 	| 2 Weeks                  	| Continuous      	| N/A                             	|                                                                                   	|
+| Miscarriage/ Pregnancy Loss Leave Leave 	| 2 Weeks                  	| Continuous      	| N/A                             	| 2 - 4 weeks is our starting point for the loss of pregnancy, we will support via additional time as needed                                                                                  	|
```

Comment:
@ClydeStackhouse can you make sure "Leave" Isn't listed twice? I can't tell in the documentation.